applications/RANSApplication/custom_strategies/relaxed_dof_updater.cpp

```diff
@@ -0,0 +1,79 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// Project includes
+#include "spaces/ublas_space.h"
+#include "utilities/parallel_utilities.h"
+
+// Include base h
+#include "relaxed_dof_updater.h"
+
+namespace Kratos
+{
+template <>
+void RelaxedDofUpdater<UblasSpace<double, CompressedMatrix, Vector>>::Initialize(
+    const DofsArrayType& rDofSet,
+    const SystemVectorType& rDx)
+{
+}
+
+template <>
+void RelaxedDofUpdater<UblasSpace<double, CompressedMatrix, Vector>>::Clear()
+{
+}
+
+template <>
+void RelaxedDofUpdater<UblasSpace<double, CompressedMatrix, Vector>>::UpdateDofs(
+    DofsArrayType& rDofSet,
+    const SystemVectorType& rDx)
+{
+    block_for_each(rDofSet, [&](DofType& rDof) {
+        if (rDof.IsFree()) {
+            rDof.GetSolutionStepValue() += rDx[rDof.EquationId()] * mRelaxationFactor;
+        }
+    });
+}
+
+template <>
+void RelaxedDofUpdater<UblasSpace<double, CompressedMatrix, Vector>>::AssignDofs(
+    DofsArrayType& rDofSet,
+    const SystemVectorType& rX)
+{
+    block_for_each(rDofSet, [&](DofType& rDof) {
+        if (rDof.IsFree()) {
+            rDof.GetSolutionStepValue() = rX[rDof.EquationId()];
+        }
+    });
+}
```

Comment:
I think this is exactly equal to the base class one.

Reply:
copy paste mistake...... removed :)